### ISSUE TITLE
Signatures 04: Typo in multiple Signatures signature string?

### ISSUE THREAD

**Issue by bblfish** [httpwg/http-extensions#1493]
The [§4.3 Multiple Signatures example](https://www.ietf.org/archive/id/draft-ietf-httpbis-message-signatures-04.html#section-4.3) has a missing quotation around `x-forwarded-for`. 

```
"signature";key="sig1": :H00a6KdNCRWgOWBMvuRtxh6c/wrVxwt2p5KyqBJqmtPbNT\
  d980hWwkUE6H4NWiTs5f2Ef0qJ3iypXT2bR9Pc+PVU9U2gAzTcZKK8MDJLjYKfaE835zg\
  /9sOdGR+tlRJ1cbCoWMVoCgEPi4t6QewbI0xgdx8AmP5ItTunYmhe8G0JR42lfvz60+sz\
  b8SpwJEmkMPr5dBOz6DLEeM3IgKNoBlJPp94WSJkgvwTM64rXw049ZkYenl9jwKlcXEmA\
  1a4MNWoUElr6eh5k20djMZftCYTPUUPMxZUavcQy+cp6lfKonz6HIDe3+n3VOTOo8uu1a\
  SVfKQQzR+ZEwSaZQBrdQ==:
x-forwarded-for: 192.0.2.123
"@signature-params": ("signature";key="sig1" x-forwarded-for)\
  ;created=1618884475;keyid="test-key-rsa";alg="rsa-v1_5-sha256"
  ```
Was this a copy-paste type from the python script? Since this is a simple RSA key, the signature generation is deterministic, so it should be easy to test.

**Comment by jricher** [comment]
Confirmed that this was an error in the example generator script, thank you! Just staging here, the string and results of the re-generated signature are:

~~~
"signature";key="sig1": \
  :YlizxWySaL8RiCQOWpl/8TBLlinl/O9K5n+WCYladKkRfmZ4wdo42ikCrepkIoPd\
  csPIx5wYc53Kpq6PLmv3fRk/+BtFSJNdrfClMEg4kX8utYuMQhBHSLiEiDRjNSTWX\
  Wk8hwutEGijbna3FvBVzy1oa5tT08w/ffN7d/6dup1FWVt90KhK1Cx3RkQveMxAKC\
  3HH6Q26lAgJ54MyLqLrLXcjQKWhjWzkMkVjho4JLy87GTU6k4eIQxB4xDHbavbJKE\
  jS0Vlg2pqmcUkGVdL4zQ3NOOttIlKC1HL1SodXNd7UBM0C0R1GGqEi4Lsm9UKWuQP\
  vPFTW7qIgvjAthv/lA==:
"x-forwarded-for": 192.0.2.123
"@signature-params": ("signature";key="sig1" "x-forwarded-for")\
  ;created=1618884480;keyid="test-key-rsa";alg="rsa-v1_5-sha256"
~~~

~~~
Signature: sig1=:YlizxWySaL8RiCQOWpl/8TBLlinl/O9K5n+WCYladKkRfmZ4wd\
    o42ikCrepkIoPdcsPIx5wYc53Kpq6PLmv3fRk/+BtFSJNdrfClMEg4kX8utYuMQ\
    hBHSLiEiDRjNSTWXWk8hwutEGijbna3FvBVzy1oa5tT08w/ffN7d/6dup1FWVt9\
    0KhK1Cx3RkQveMxAKC3HH6Q26lAgJ54MyLqLrLXcjQKWhjWzkMkVjho4JLy87GT\
    U6k4eIQxB4xDHbavbJKEjS0Vlg2pqmcUkGVdL4zQ3NOOttIlKC1HL1SodXNd7UB\
    M0C0R1GGqEi4Lsm9UKWuQPvPFTW7qIgvjAthv/lA==:, \
  proxy_sig=:FSqkfwt17TLKqjjWrW9vf6a/r3329amAO3e7ByjkT60jjFTq4xdO74\
    JTHrpz6DMSlQOKmhIiz8mq7T5SYOjfUZrKXpbP6jUFTStUa4HvNNvjhZc1jiHk9\
    IhGGPPeOdRcTrzjDxSS+2l7G3nSpJ4t2LjtLnEPa1FIldgnJqwIa0SCiEPWFmnJ\
    fTdc4VW2ngvYhuKUKFz/Jyx0GfmKQ4lAWxQwVtqRSURTscdY3VvxR+GpydqF8gQ\
    U8iYslRHlRxBh+29cADAHVnT5j1iBkVdAfLS59xCYLnUc3UG7UfxU6kU1QgJ+2n\
    A5NNQsxXeREcCFTe2FnjOy2atxG8bm+O8ZcA==:
~~~

(note that the input signature is non-deterministic and was re-calculated by the tools)

**Comment by bblfish** [comment]
Great: I am able to verify the new signature on the newly provided input string.

**Comment by jricher** [comment]
Fixed in #1528
